packages/nvidia_nat_core/src/nat/front_ends/fastapi/routes/auth.py

```
@@ -0,0 +1,77 @@
+        try:
+            res = await client.fetch_token(url=config.token_url,
+                                           authorization_response=str(request.url),
+                                           code_verifier=verifier,
+                                           state=state)
+            flow_state.future.set_result(res)
+        except OAuthError as e:
+            flow_state.future.set_exception(
+                RuntimeError(f"Authorization server rejected request: {e.error} ({e.description})"))
+        except httpx.HTTPError as e:
+            flow_state.future.set_exception(RuntimeError(f"Network error during token fetch: {e}"))
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)